x-pack/plugins/security_solution/common/detection_engine/schemas/request/create_rules_type_dependents.test.ts

```diff
@@ -4,31 +4,11 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import {
-  getCreateRulesSchemaMock,
-  getCreateThreatMatchRulesSchemaMock,
-} from './create_rules_schema.mock';
-import { CreateRulesSchema } from './create_rules_schema';
+import { getCreateRulesSchemaMock, getCreateThreatMatchRulesSchemaMock } from './rule_schemas.mock';
+import { CreateRulesSchema } from './rule_schemas';
 import { createRuleValidateTypeDependents } from './create_rules_type_dependents';
 
 describe('create_rules_type_dependents', () => {
-  test('saved_id is required when type is saved_query and will not validate without out', () => {
```

Comment:
These are now covered by static type checks.

Reply:
Make sure these are covered in schema tests?

Reply:
:+1: added tests covering the same behavior in `rule_schemas.test.ts`